build.gradle.kts

```diff
@@ -174,12 +174,17 @@ tasks {
                 "RedundantOverride",
                 "RedundantThrows",
                 "RemoveUnusedImports",
+                "DefaultCharset",
                 "UnnecessarilyFullyQualified",
                 "UnnecessarilyUsedValue",
                 "UnnecessaryBoxedAssignment",
                 "UnnecessaryBoxedVariable",
                 "UnnecessaryFinal",
                 "UnusedException",
+                "UnusedLabel",
```

Comment:
Where did you find these ones all of a sudden? :-D

Reply:
Mostly looking at them all :D 
https://errorprone.info/bugpatterns